setup.py

```diff
@@ -1,25 +1,11 @@
 import os
-import re
-from setuptools import setup, find_packages
+from setuptools import setup
 
 
 def read(f):
     return open(os.path.join(os.path.dirname(__file__), f)).read().strip()
 
 
-def read_version():
-    regexp = re.compile(r"^__version__\W*=\W*'([\d.abrc]+)'")
-    init_py = os.path.join(os.path.dirname(__file__),
-                           'aiomysql', '__init__.py')
-    with open(init_py) as f:
-        for line in f:
-            match = regexp.match(line)
-            if match is not None:
-                return match.group(1)
-        else:
-            raise RuntimeError('Cannot find version in aiomysql/__init__.py')
-
-
-setup(version=read_version(),
```

Comment:
This could be implemented as `version = attr: aiomysql.__version__` in `setup.cfg`, in a dedicated PR that just drops `setup.py`. And setuptools-scm integration could be done in a follow-up. I just feel like there's a lot of things mixed together in this patch which makes it harder to reason about the effect of the changes.

Reply:
thanks, implemented in #746